### PR TITLE
Create starkblaze01.json

### PR DESCRIPTION
**Pull request by starkblaze01**

### CODE REVIEW

File: domains/starkblaze01.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "starkblaze01's world",
+  "repo": "https://github.com/starkblaze01/starkblaze01",
+  "owner": {
+    "username": "starkblaze01",
+    "email": "mp.pathela@gmail.com"
+  },
+  "record": {
+    "CNAME": "https://starkblaze01.netlify.app/"
```
Comment (phenax) [inline domains/starkblaze01.json:9]: CNAME has to be a hostname

```suggestion
    "CNAME": "starkblaze01.netlify.app"
```

Comment (phenax) [reply domains/starkblaze01.json:9]: Also, please confirm that you can configure custom domains with netlify

**Review by phenax — CHANGES_REQUESTED** [review]

File: domains/starkblaze01.json (hunk 1 above)
Comment (starkblaze01) [reply domains/starkblaze01.json:9]: Hi, will changing "CNAME" to URL for redirect will do?
Yes, we can configure custom domains with netlify.

Comment (phenax) [reply domains/starkblaze01.json:9]: If you want to change it to a URL redirection then you don't need to configure anything in netlify.

Comment (starkblaze01) [reply domains/starkblaze01.json]: Okay, thanks. Updated the patch.

**Review by phenax — APPROVED** [review]

**Pull request merged**